README.md

```diff
@@ -9,7 +9,7 @@ pdfRest is a REST API Toolkit for developers with all of the API Tools you'll ne
 
 |[Compress PDF](https://pdfrest.com/apitoolkit/compress-pdf/) | [Convert to PDF](https://pdfrest.com/apitoolkit/convert-to-pdf/) | [Merge PDFs](https://pdfrest.com/apitoolkit/merge-pdfs/) | [Encrypt PDF](https://pdfrest.com/apitoolkit/encrypt-pdf/) | [Restrict PDF](https://pdfrest.com/apitoolkit/restrict-pdf/) | [Flatten Transparencies](https://pdfrest.com/apitoolkit/flatten-transparencies/) | [Zip Files](https://pdfrest.com/apitoolkit/zip-files/)
 |:-|:-|:-|:-|:-|:-|:-|
-__[PDF to Images](https://pdfrest.com/apitoolkit/pdf-to-images/)__ | __[Convert to PDF/A](https://pdfrest.com/apitoolkit/convert-to-pdfa/)__ | __[Split PDF](https://pdfrest.com/apitoolkit/split-pdf/)__ | __[Decrypt PDF](https://pdfrest.com/apitoolkit/encrypt-pdf/)__ | __[Unrestrict PDF](https://pdfrest.com/apitoolkit/restrict-pdf/)__ | __[Linearize PDF](https://pdfrest.com/apitoolkit/linearize-pdf/)__ | __[Query PDF](https://pdfrest.com/apitoolkit/query-pdf/)__ |
+__[PDF to Images](https://pdfrest.com/apitoolkit/pdf-to-images/)__ | __[Convert to PDF/A](https://pdfrest.com/apitoolkit/convert-to-pdfa/)__ | __[Split PDF](https://pdfrest.com/apitoolkit/split-pdf/)__ | __[Decrypt PDF](https://pdfrest.com/apitoolkit/encrypt-pdf/)__ | __[Unrestrict PDF](https://pdfrest.com/apitoolkit/restrict-pdf/)__ | __[Linearize PDF](https://pdfrest.com/apitoolkit/linearize-pdf/)__ | __[Query PDF](https://pdfrest.com/apitoolkit/query-pdf/)__ | __[Add to PDF](https://pdfrest.com/apitoolkit/add-to-pdf/)__ |
```

Comment:
I don't see **Add to PDF** in the table when I _View file_.

Reply:
It looks like the last time there was a mismatching number of table rows, there were two bars instead of one at the end. Looks like `| |`. (See https://github.com/datalogics/pdf-rest-api-samples/commit/77e318d5ca9bf7f735b10355af46eaa4899dc043 and look at the original.)

Reply:
```suggestion
__[PDF to Images](https://pdfrest.com/apitoolkit/pdf-to-images/)__ | __[Convert to PDF/A](https://pdfrest.com/apitoolkit/convert-to-pdfa/)__ | __[Split PDF](https://pdfrest.com/apitoolkit/split-pdf/)__ | __[Decrypt PDF](https://pdfrest.com/apitoolkit/encrypt-pdf/)__ | __[Unrestrict PDF](https://pdfrest.com/apitoolkit/restrict-pdf/)__ | __[Linearize PDF](https://pdfrest.com/apitoolkit/linearize-pdf/)__ | __[Query PDF](https://pdfrest.com/apitoolkit/query-pdf/)__ | __[Add to PDF](https://pdfrest.com/apitoolkit/add-to-pdf/)__ | |
```

Reply:
> It looks like the last time there were mismatched numbers of table columns, there were two bars instead of one at the end. Looks like `| |`. (See [77e318d](https://github.com/datalogics/pdf-rest-api-samples/commit/77e318d5ca9bf7f735b10355af46eaa4899dc043) and look at the original.)

Never mind -- this didn't work for me, either! The double bars probably added a blank cell where there was room for one in that row.

The actual solution may be to begin a new row in the table, but mine's coming out gray for some reason, which doesn't seem ideal.